src/expandableCalendar/WeekCalendar/index.js

```diff
@@ -102,79 +102,46 @@ class WeekCalendar extends Component {
   });
 
   onDayPress = value => {
-    _.invoke(this.props.context, 'setDate', value.dateString, UPDATE_SOURCES.DAY_PRESS);
+    this.presenter.onDayPressed(this.props.context, value);
   };
 
   onScroll = ({
     nativeEvent: {
       contentOffset: {x}
     }
   }) => {
-    if (!this.firstAndroidRTLScrollIgnored) {
-      this.firstAndroidRTLScrollIgnored = true;
-      return;
-    }
-
-    // Fix reversed offset on Android+RTL
-    if (applyAndroidRtlFix) {
-      const numOfPages = this.state.items.length - 1;
-      const overallWidth = numOfPages * this.containerWidth;
-      x = overallWidth - x;
-    }
-
-    const newPage = Math.round(x / this.containerWidth);
+    const {onScroll} = this.presenter;
+    const {context} = this.props;
+    const {items} = this.state;
+    const {containerWidth: width, page} = this;
```

Comment:
Why the renaming of 'containerWidth' is necessary here? Isn't it more readable to pass it as is (width: containerWidth) to the presenter's onScroll()?

Reply:
Ok